### ISSUE TITLE
Document provenance of reference atomic configurations

### ISSUE THREAD

**Issue by max-veit** [cosmo-epfl/librascal#226]
Spun off from #181 (see [here](https://github.com/cosmo-epfl/librascal/pull/181#discussion_r339996147)): We have many atomic configurations just called e.g. `methane.xyz` and `H2O.xyz`, and it might be useful to know how the geometry itself was obtained (from a database, or optimized, and if so, with which method?). These can be added as a comment in the (extended) XYZ file itself, by adding a tag like `comment="from GDB9, re-optimized with B3LYP/6-31G*"` to the comment line).

**Comment by max-veit** [comment]
(BTW for json-formatted reference configurations it's even easier to add a comment field. Just add a new entry labelled "comment" and all our readers should ignore it.)

**Comment by felixmusil** [comment]
> (BTW for json-formatted reference configurations it's even easier to add a comment field. Just add a new entry labelled "comment" and all our readers should ignore it.)

I agree with you the json format is perfect for that and it is more convenient to have self documenting files.

**Comment by mastricker** [comment]
Should everyone change her/his contributions in #181 or how will we handle this?

**Comment by rosecers** [comment]
> Should everyone change her/his contributions in #181 or how will we handle this?

I think it would be best if everyone took a bit and identified which data files came from them or that 
they know the provenance of. I'll list the files in reference_data/* here, and please comment if you can take responsibility for a file and then check it off.

inputs:

- [ ] 1-Propanol.xyz
- [ ] 2-Propanol.xyz
- [ ] alanine-center-select.json
- [ ] alanine-X-examples.json
- [ ] alanine-X.json
- [ ] benzene.xyz
- [ ] CaCrP2O7_mvc-11955_symmetrized.json
- [ ] crystal_structure.json
- [ ] dft-smiles_500.xyz
- [ ] diamond_2atom_distorted.json
- [ ] diamond_2atom.json
- [ ] diamond_cubic_distorted.json
- [ ] diamond_cubic.json
- [ ] dummy_structure.json
- [ ] dummy_structure_wrapped.json
- [ ] H2O.xyz
- [ ] mbfs_derivative_test.json
- [ ] methane.json
- [ ] methane.xyz
- [ ] Methoxyethane.xyz
- [ ] modified_bessel_first_kind_reference.ubjson
- [ ] molecular_crystal.json
- [ ] N3.xyz
- [ ] polyalanine.json
- [ ] S6.xyz
- [ ] SiCGe_wurtzite_like.json
- [ ] SiC_moissanite.json
- [ ] SiC_moissanite_supercell.json
- [ ] simple_cubic_8.json
- [ ] simple_cubic_9.json
- [ ] small_molecule.json
- [ ] small_molecule_no_cell.json
- [ ] small_molecules-1000.xyz
- [ ] water_rotations.xyz

tests_only:

- [ ] cutoff_function_test.json
- [ ] dft-smiles_500.ubjson
- [ ] gauss_legendre_reference.ubjson
- [ ] hyp1f1_reference.ubjson
- [ ] kernel_reference.ubjson
- [ ] modified_bessel_first_kind_reference.json
- [ ] radial_derivative_test.json
- [ ] sorted_coulomb_reference.ubjson
- [ ] spherical_covariants_reference.ubjson
- [ ] spherical_expansion_gradient_test.json
- [ ] spherical_expansion_reference.ubjson
- [ ] spherical_harmonics_gradient_test.json
- [ ] spherical_harmonics_reference.ubjson
- [ ] spherical_harmonics_test.json
- [ ] spherical_invariants_reference.ubjson

unused:

- [ ] behler_parinello_pair_hypers.json
- [ ] CaCrP2O7_mvc-11955_symmetrized.cif
- [ ] primitive_cubic.json
- [ ] simple_cubic_3.json

**Comment by mastricker** [comment]
And directly change it in #230  ?

**Comment by rosecers** [comment]
> And directly change it in #230 ?

I'd like to merge in #230 sooner rather than later. I think right now, the first goal is to have each file claimed, and then we can create a separate branch/PR for labeling. Thoughts?

**Comment by max-veit** [comment]
The `methane.{xyz,json}` were from me (surprise, surprise); I think they were just the ASE G2-database structures, but I'll have to check. I think I also added the diamond, moissanite, and wurtzite structures for gradient testing based off of cell parameters from Wikipedia.

But yes, let's merge the PR first and then add these as we go along in another branch.

**Comment by rosecers** [comment]
> The `methane.{xyz,json}` were from me (surprise, surprise); I think they were just the ASE G2-database structures, but I'll have to check. I think I also added the diamond, moissanite, and wurtzite structures for gradient testing based off of cell parameters from Wikipedia.
> 
> But yes, let's merge the PR first and then add these as we go along in another branch.

Max, can you check off the files that belong to you?

**Comment by mastricker** [comment]
Mine, according to "user" tag are
```
./unused/simple_cubic_3.json:    "user": "markus"},
./inputs/simple_cubic_9.json:    "user": "markus"},
./inputs/alanine-center-select.json: "user": "markus"},
./inputs/alanine-X-examples.json: "user": "markus"},
./inputs/polyalanine.json: "user": "markus stricker"},
./inputs/crystal_structure.json:    "user": "markus stricker"},
./inputs/alanine-X.json: "user": "markus"},
```
I am writing them here for reference.

Should everybody do her/his own PR for the metadata change?

**Comment by mastricker** [comment]
Draft PR branch open every file-claimer at #236
